browser-test/src/phone.test.ts

```diff
@@ -137,14 +137,28 @@ describe('phone question for applicant flow', () => {
         'This phone number is invalid',
       )
     })
-    it('invalid characters in phone numbers', async () => {
+    it('invalid length of phone numbers', async () => {
```

Comment:
Question (blocking): My initial thought was that since this test is only for invalid length not invalid characters, maybe we should remove the invalid characters so that this test case is just focused on length. But then I noticed that line 168 also has a test case for invalid length, so now I'm not quite sure what this is testing.

I'm wondering: Is this test different in some way from the test case on line 168? Is this testing a phone number has the right length *before* removing invalid characters but the wrong length *after* removing the invalid characters? If so, can we name this test case something clearer like 'invalid length of phone number when only valid characters included'?

(Feel free to word-smith that test name, I think we just need to make it clear how this test is different from the test on line 168.)